db/warehouse/migrate/20230803173117_migrate_project_type_form_instance.rb

```diff
@@ -0,0 +1,6 @@
+class MigrateProjectTypeFormInstance < ActiveRecord::Migration[6.1]
+  def up
+    Hmis::Form::Instance.where(entity_type: 'ProjectType').update_all('project_type=entity_id')
```

Comment:
```suggestion
    Hmis::Form::Instance.where(entity_type: 'ProjectType').update_all(project_type: 'entity_id')
```

Reply:
Wouldn't that update the project type column to be the string `'entity_id'`? What I'm trying to do is actually move over the value from the entity_id column